Library/Formula/apcupsd.rb

```diff
@@ -0,0 +1,84 @@
+class Apcupsd < Formula
+  desc "Daemon for controlling APC UPSes"
+  homepage "http://www.apcupsd.org"
+  url "https://downloads.sourceforge.net/project/apcupsd/apcupsd%20-%20Stable/3.14.13/apcupsd-3.14.13.tar.gz"
+  sha256 "57ecbde01d0448bf8c4dbfe0ad016724ae66ab98adf2de955bf2be553c5d03f9"
+
+  option "with-cgi", "Compile CGI programs"
+  option "with-usb", "Compile USB UPS code"
+  option "with-modbus-usb", "Compile MODBUS/USB driver code"
+
+  depends_on "libusb-compat"
+  depends_on "gd" if build.with? "cgi"
+
+  def install
+    cd "src/apcagent" do
+      # Install apcagent.app to `prefix` instead of /Applications.
+      inreplace "Makefile", "Applications", "#{"#{prefix}".gsub(%r{/^\//}, "")}"
+    end
+
+    sysconfdir = "#{etc}/apcupsd"
+
+    cd "platforms/darwin" do
+      inreplace "Makefile", "/sbin", sbin
+      inreplace "Makefile", "/Library/LaunchDaemons", "#{prefix}/Library/LaunchDaemons"
+      inreplace "Makefile", "/System/Library/Extensions", kext_prefix
+
+      inreplace "apcupsd-start", "/sbin", opt_sbin
+      inreplace "apcupsd-start", "/etc/apcupsd", sysconfdir
+
+      inreplace "apcupsd-uninstall", "/Library/StartupItems/apcupsd", "#{prefix}.split('/')[0..-2].join('/')"
+      inreplace "apcupsd-uninstall", "/sbin", sbin
+      inreplace "apcupsd-uninstall", "/etc/apcupsd", sysconfdir
+      inreplace "apcupsd-uninstall", "/usr/share/man", man
+
+      inreplace "org.apcupsd.apcupsd.plist", "/sbin", opt_sbin
+      inreplace "org.apcupsd.apcupsd.plist", "/etc/apcupsd", sysconfdir
+    end
+
+    args = [
+      "--disable-option-checking",
+      "--prefix=#{prefix}",
+      "--sbindir=#{sbin}",
+      "--sysconfdir=#{sysconfdir}",
+      "--with-cgi-bin=#{sysconfdir}",
+    ]
+
+    args << "--enable-cgi" if build.with? "cgi"
+    args << "--enable-usb" if build.with? "usb"
+    args << "--enable-modbus-usb" if build.with? "modbus-usb"
+
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  def caveats
+    <<-EOS.undent
+      The apcupsd configuration files are located here:
+        #{etc}/apcupsd
+
+      In order for apcupsd to work, the apcupsd kernel extension
+      must be installed by the root user:
+
+        sudo cp -pR #{kext_prefix}/ApcupsdDummy.kext /System/Library/Extensions/
+        sudo chown -R root:wheel /System/Library/Extensions/ApcupsdDummy.kext
+        sudo touch /System/Library/Extensions/
```

Comment:
I also found this: https://forums.developer.apple.com/thread/19553
Unfortunately, there's no response.

It would be interesting to know if in El Capitan `AppleKextExcludeList.kext` is ignored or not.